lib/katello/tasks/annotate_scenarios.rake

```diff
@@ -0,0 +1,41 @@
+require "#{Katello::Engine.root}/test/support/annotation_support"
+namespace :katello do
+  ANNOTATION_DIR = "#{Katello::Engine.root}/test/scenarios/annotations/".freeze
+  CASSETTE_DIR = "#{Katello::Engine.root}/test/fixtures/vcr_cassettes/scenarios/".freeze
+  OUTPUT_DIR = "#{Rails.root}/tmp/".freeze
+
+  desc ""
```

Comment:
Were you intending to have a description? Makes it easier to find if you forget the name with `rake -T`